Sources/Basics/Cancellator.swift

```diff
@@ -54,9 +54,11 @@ public class Cancellator: Cancellable {
         self.register(name: "\(process.arguments.joined(separator: " "))", handler:  process.terminate)
     }
 
+    #if !os(iOS) && !os(watchOS) && !os(tvOS)
```

Comment:
I'd prefer if there were a better way to do this, but I don't think there's a `#if defined(Foundation.Process)` or anything like that.

Reply:
does `#if canImport(Foundation.Process)` work?

Reply:
Unfortunately not: `Unexpected platform condition argument: expected identifier`.  That would have been great!

Maybe this is worth a request on the compiler.  At the least the error seems wrong — it's not a platform condition.